Resources/Maps/Shuttles/Mushroom.yml

```diff
@@ -0,0 +1,5827 @@
+    - rot: -1.5707963267948966 rad
+      pos: 2.5,-2.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 460
+      type: DeviceNetwork
+  - uid: 464
+    components:
+    - pos: -1.5,-9.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 459
+      type: DeviceNetwork
+  - uid: 465
+    components:
+    - pos: -0.5,-9.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 459
+      type: DeviceNetwork
+- proto: FirelockEdge
+  entities:
+  - uid: 466
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -3.5,-12.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 458
+      type: DeviceNetwork
+  - uid: 467
+    components:
+    - pos: -1.5,-4.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 459
+      type: DeviceNetwork
+  - uid: 468
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 0.5,-7.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 459
+      type: DeviceNetwork
+  - uid: 469
+    components:
+    - pos: -0.5,-4.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 459
+      type: DeviceNetwork
+  - uid: 470
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 1.5,-12.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 458
+      type: DeviceNetwork
+- proto: FirelockGlass
+  entities:
+  - uid: 471
+    components:
+    - pos: -1.5,-20.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 458
+      type: DeviceNetwork
+  - uid: 472
+    components:
+    - pos: -0.5,-20.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 458
+      type: DeviceNetwork
+  - uid: 473
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: 4.5,-4.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 460
+      type: DeviceNetwork
+  - uid: 474
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -6.5,-4.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 460
+      type: DeviceNetwork
+  - uid: 475
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: -1.5,-2.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 460
+      type: DeviceNetwork
+  - uid: 476
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: -0.5,-2.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 460
+      type: DeviceNetwork
+  - uid: 477
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -6.5,-5.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 460
+      type: DeviceNetwork
+  - uid: 478
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: 4.5,-5.5
+      parent: 1
+      type: Transform
+    - ShutdownSubscribers:
+      - 460
+      type: DeviceNetwork
+- proto: FloorDrain
```

Comment:
Drain in a merc and medical ship?

Reply:
well the sprinter has one including the dart, anchor even tho the dart and anchor isnt an merc ship

Reply:
Drain has been removed